content/en/docs/collector/transforming-telemetry.md

```diff
@@ -169,6 +169,31 @@ processors:
   k8sattributes/default:
 ```
 
+## Setting a span status
+
+**Processor**:
+[transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor)
+
+Use the transform processor to set a span's status. The following example sets
+the span status to `Ok` when the `http.status_code` attribute is 400:
+
+```yaml
+# The description field allows to set a human-readable message for errors.
+transform:
+  error_mode: ignore
+  trace_statements:
+    - context: span
+      statements:
+        - set(status.code, STATUS_CODE_OK) where
+          attributes["http.request.status_code"] == 400
+```
+
+You can also use the transform processor to modify the span name based on its
+attributes or extract span attributes from the span name. For examples, see
+`transform processor` test data
+[config.yaml](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/transformprocessor/testdata/config.yaml)
+file.
```

Comment:
```suggestion
attributes or extract span attributes from the span name. For examples, see an example
[config file](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/9b28f76c02c18f7479d10e4b6a95a21467fd85d6/processor/transformprocessor/testdata/config.yaml)
file for the transform processor.
```
Mostly just minor rewording, feel free to change this if you think it could be improved.

Also, I think we should pin this link to an exact commit. As it stands, if this file is moved in the contrib repo on the main branch, the link will break. I found a commit that should work, but another commit would work as well.

Reply:
Ideally we point people to the most recent version of that file. @chalin if that file gets moved, are we getting informed about that with a 404? I don't think that we are doing that right now.

Reply:
We aren't currently checking and then caching links into GH repos. Do we want to start doing that now?

Reply:
I think especially for links to other otel resources this would be great to have, I am not that worried about outside sources (yet), if this makes things easier in any way

Reply:
I created the following issue to track the additional checking:

- #3994

Reply:
@Gaurang-Patel please apply this change